src/components/notebook/EmptyStateCellAdder.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Button } from "@/components/ui/button";
+import { Bot, Code, Database, FileText } from "lucide-react";
+
+interface EmptyStateCellAdderProps {
+  onAddCell: (
+    cellId?: string,
+    cellType?: "code" | "markdown" | "sql" | "ai",
+    position?: "before" | "after"
+  ) => void;
+}
+
+export const EmptyStateCellAdder: React.FC<EmptyStateCellAdderProps> = ({
+  onAddCell,
+}) => {
+  return (
+    <div className="px-4 pt-6 pb-6 text-center sm:px-0 sm:pt-12">
+      <div className="text-muted-foreground mb-6">
+        Welcome to your notebook! Choose a cell type to get started.
+      </div>
+      <div className="mb-4 flex flex-wrap justify-center gap-2">
+        <Button
+          size="lg"
+          autoFocus
+          onClick={() => onAddCell()}
+          className="flex items-center gap-2"
+        >
+          <Code className="h-4 w-4" />
+          Code Cell
+        </Button>
+        <Button
+          size="lg"
+          variant="outline"
+          color="yellow"
+          onClick={() => onAddCell(undefined, "markdown")}
+          className="flex items-center gap-2"
+        >
+          <FileText className="h-4 w-4" />
+          Markdown
+        </Button>
+        <Button
+          size="lg"
+          variant="outline"
+          color="blue"
+          onClick={() => onAddCell(undefined, "sql")}
+          className="flex items-center gap-2"
+        >
+          <Database className="h-4 w-4" />
+          SQL Query
+        </Button>
+        <Button
+          size="lg"
+          variant="outline"
+          color="purple"
```

Comment:
I feel like we should be pulling these colors from a similar spot for the cells. :-/